modules/mssqlserver/src/main/java/org/testcontainers/containers/MSSQLServerContainer.java

```diff
@@ -0,0 +1,58 @@
+package org.testcontainers.containers;
+
+/**
+ * @author Stefan Hufschmidt
+ */
+public class MSSQLServerContainer<SELF extends MSSQLServerContainer<SELF>> extends JdbcDatabaseContainer<SELF> {
+    static final String NAME = "mssqlserver";
+    static final String IMAGE = "microsoft/mssql-server-linux";
+    public static final Integer MS_SQL_SERVER_PORT = 1433;
+    private String username = "SA";
+    private String password = "A_Str0ng_Required_Password";
+
+    public MSSQLServerContainer() {
+        this(IMAGE + ":latest");
+    }
+
+    public MSSQLServerContainer(final String dockerImageName) {
+        super(dockerImageName);
+    }
+
+    @Override
+    protected Integer getLivenessCheckPort() {
+        return getMappedPort(MS_SQL_SERVER_PORT);
+    }
+
+    @Override
+    protected void configure() {
+
+        addExposedPort(MS_SQL_SERVER_PORT);
+        addEnv("ACCEPT_EULA", "Y");
```

Comment:
I wonder if it's legal or not, to set it automatically. We might have to make it explicit to avoid legal cases with M$

Reply:
Very good point - I've added a simple license check mechanism (see new `LicenseAcceptance` class). Acceptance is by adding a file to the classpath, so hopefully not too much of a burden.

My initial plan was to add an 'acceptEula' parameter to the constructor, but that falls apart when it comes to JDBC URL usage.